charmcraft.yaml

```diff
@@ -2,24 +2,28 @@
 # See LICENSE file for licensing details.
 name: cephfs-server-proxy
 type: charm
-
 title: CephFS Proxy
-
 summary: Charmed Ceph FS server for non-charmed Ceph storage appliances.
-
-
 description: |
   Charmed Ceph FS server for non-charmed Ceph storage appliances.
 
   Enable CephFS Client charms to mount non-charmed Ceph shares.
-
-bases:
-  - build-on:
-    - name: ubuntu
-      channel: "22.04"
-    run-on:
-    - name: ubuntu
-      channel: "22.04"
+links:
+  source: 
+  - https://github.com/canonical/cephfs-server-proxy-operator
+  issues: 
+  - https://github.com/canonical/cephfs-server-proxy-operator/issues
+
+base: ubuntu@24.04
+platforms:
+  amd64:
+
+parts:
+  charm: {}
```

Comment:
IIRC you don't need to manually declare the `charm` part here since we don't have any overrides or other parts being built. If we added custom parts, then we would need the blank charm `charm: {}` specified:

```suggestion
```

Reply:
Tried that, but charmcraft apparently chokes in this case and doesn't build any files. Probably a bug.

Reply:
That's... not good. We'll leave it in then since it just makes the `charm` part explicit rather than implicit.

Reply:
> Tried that, but charmcraft apparently chokes in this case and doesn't build any files. Probably a bug.

Make sure you report this btw. You could just drop a message in the charmcraft room on Matrix: https://matrix.to/#/#charmhub-charmcraft:ubuntu.com

Reply:
It is a known issue.
https://github.com/canonical/charmcraft/issues/2055